### PR TITLE
Fixing https://wso2.org/jira/browse/APIMANAGER-5080

### PR DESCRIPTION
**Pull request by dewmini**

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/life-cycles/template.jag [hunk 1]
```diff
@@ -87,7 +88,52 @@ jagg.message({
...
+            jagg.post("/site/blocks/life-cycles/ajax/life-cycles.jag", {
+                   action: "updateStatus",
+                   provider: provider,
+                   name: apiName,
+                   version: version,
+                   status: status
+            }, function (result) {
+                if (!result.error) {
+                    $('#statusUpdateMsg').show();
+                    setTimeout(function(){
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/life-cycles/template.jag]: Formatting issue. Please add spacing as relevant.

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/life-cycles/template.jag [hunk 2]
```diff
@@ -87,7 +88,52 @@ jagg.message({
...
+                } else {
+                    if (result.message == "AuthenticateError") {
+                        jagg.showLogin();
+                    } else {
+                        jagg.message({content:result.message,type:"error"});
+                    }
+                  }
+            }, "json");
+        },
+        cancelCallback:function(){
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/life-cycles/template.jag]: Formatting issue. Please add spacing as relevant.

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/life-cycles/template.jag (hunk 1 above)
Comment (dewmini) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/life-cycles/template.jag]: Fixed in https://github.com/wso2/carbon-apimgt/commit/e3c49af10e8752b1b6531c3fabea1c1c56e1330d

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/life-cycles/template.jag (hunk 2 above)
Comment (dewmini) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/life-cycles/template.jag]: Fixed in https://github.com/wso2/carbon-apimgt/commit/e3c49af10e8752b1b6531c3fabea1c1c56e1330d

**Pull request merged**